app/controllers/members/dues_controller.rb

```diff
@@ -5,6 +5,7 @@ def show
     if current_user.stripe_customer_id
       customer = Stripe::Customer.retrieve(current_user.stripe_customer_id)
       @subscription = customer.subscriptions.first
+      @current_plan = plans[@subscription.plan.amount]
```

Comment:
I am not sure it's guaranteed that the `plan.amount` will be in that list (because of legacy reasons, I think it might be different), so I think we should instead use `fetch` with a `nil` default:
```ruby
@current_plan = plans.fetch(@subscription.plan.amount, nil)
```